sass/breadcrumbs.scss

```diff
@@ -1,25 +1,32 @@
-.castlecss-breadcrumbs{
-	
-	.breadcrumb-wrap {
-		display: block;		
+.castlecss-breadcrumbs {
+	font-size: 1.4rem;
+}	
 
-		.breadcrumb {
-			display: inline;
-			white-space: nowrap;
+.breadcrumbs-wrap {
+	display: block;	
+}	
 
-			&:after {
-				display: inline;
-				content: '\00bb';
-				color: $theme01;
-			}
-			&.breadcrumb-current {
-				
-				&:after {
-					content:"";
-				}
-			}
+.breadcrumb {
+	display: inline;
+	white-space: nowrap;
+
+	&:after {
+		display: inline;
+		
+		@if variable-exists(castlecss-breadcrumbs-separator) {
+			content: $castlecss-breadcrumbs-separator; // default separator character between crumbs.
```

Comment:
Agreed, adding castlecss- to the variables seems a bit overkill to me. We're not doing it for other variables neither, lets leave it out?